persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -145,8 +153,10 @@ public static void assertAuxiliaryFunctions() {
             persistCmd.printLongDesc(cmdLongDesc);
             persistCmd.printUsage(cmdLongDesc);
             String cmdLongDescription = cmdLongDesc.toString();
-            Assert.assertEquals(initLongDescription.trim().replaceAll(System.lineSeparator(), ""),
-                    "Generate database configurations file inside the Ballerina project  ballerina persist init");
+            Assert.assertEquals(initLongDescription.trim()
+                            .replaceAll(System.lineSeparator(), ""),
+                    "Generate database configurations file inside " +
+                            "the Ballerina project  ballerina persist init");
```

Comment:
Shall we keep the existing code as it is?
```suggestion
            Assert.assertEquals(initLongDescription.trim().replaceAll(System.lineSeparator(), ""),
                    "Generate database configurations file inside the Ballerina project  ballerina persist init");
```